firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/internal/util/conversions.kt

```diff
@@ -16,92 +16,13 @@
-import com.google.firebase.vertexai.type.FunctionCallingConfig
-import com.google.firebase.vertexai.type.FunctionDeclaration
-import com.google.firebase.vertexai.type.GenerateContentResponse
-import com.google.firebase.vertexai.type.GenerationConfig
-import com.google.firebase.vertexai.type.HarmBlockMethod
-import com.google.firebase.vertexai.type.HarmBlockThreshold
-import com.google.firebase.vertexai.type.HarmCategory
-import com.google.firebase.vertexai.type.HarmProbability
-import com.google.firebase.vertexai.type.HarmSeverity
-import com.google.firebase.vertexai.type.ImagePart
-import com.google.firebase.vertexai.type.InlineDataPart
-import com.google.firebase.vertexai.type.Part
-import com.google.firebase.vertexai.type.PromptFeedback
-import com.google.firebase.vertexai.type.SafetyRating
-import com.google.firebase.vertexai.type.SafetySetting
 import com.google.firebase.vertexai.type.SerializationException
-import com.google.firebase.vertexai.type.TextPart
-import com.google.firebase.vertexai.type.Tool
-import com.google.firebase.vertexai.type.ToolConfig
-import com.google.firebase.vertexai.type.UsageMetadata
-import com.google.firebase.vertexai.type.content
-import java.io.ByteArrayOutputStream
-import java.util.Calendar
 import kotlinx.serialization.json.Json
-import kotlinx.serialization.json.JsonNull
 import kotlinx.serialization.json.JsonObject
 import org.json.JSONObject
 
-private const val BASE_64_FLAGS = Base64.NO_WRAP
-
-internal fun Content.toInternal() =
-  com.google.firebase.vertexai.common.shared.Content(
-    this.role ?: "user",
-    this.parts.map { it.toInternal() }
-  )
-
-internal fun Part.toInternal(): com.google.firebase.vertexai.common.shared.Part {
-  return when (this) {
-    is TextPart -> com.google.firebase.vertexai.common.shared.TextPart(text)
-    is ImagePart ->
-      com.google.firebase.vertexai.common.shared.InlineDataPart(
-        InlineData("image/jpeg", encodeBitmapToBase64Png(image))
-      )
-    is InlineDataPart ->
-      com.google.firebase.vertexai.common.shared.InlineDataPart(
-        InlineData(mimeType, Base64.encodeToString(inlineData, BASE_64_FLAGS))
-      )
-    is com.google.firebase.vertexai.type.FunctionCallPart ->
-      FunctionCallPart(FunctionCall(name, args))
-    is com.google.firebase.vertexai.type.FunctionResponsePart ->
-      FunctionResponsePart(FunctionResponse(name, response))
-    is FileDataPart ->
-      com.google.firebase.vertexai.common.shared.FileDataPart(
-        FileData(mimeType = mimeType, fileUri = uri)
-      )
-    else ->
-      throw SerializationException(
-        "The given subclass of Part (${javaClass.simpleName}) is not supported in the serialization yet."
-      )
-  }
-}
-
-internal fun SafetySetting.toInternal() =
-  com.google.firebase.vertexai.common.shared.SafetySetting(
-    harmCategory.toInternal(),
-    threshold.toInternal(),
-    method?.toInternal()
-  )
+internal const val BASE_64_FLAGS = Base64.NO_WRAP
```

Comment:
Is this used in several places? could it be moved somewhere else?

Reply:
I moved it to the file where it's used